18F/C2
# [Delivers #108771232] tier one delegate authz

Story: https://www.pivotaltracker.com/story/show/108771232

This PR creates a new column `completed_by_id` on the `Step` model.

Testing:

* create a proposal with an approver that has two or more delegates.
* approve the proposal as a delegate
* verify you can still view the proposal as one of the other delegates
* verify that the approval records the delegate as the person who approved
